### PR TITLE
Add live stream ABR ladder profile

### PR DESCRIPTION
**Pull request by elv-serban**

### CODE REVIEW

File: src/client/LiveConf.js [hunk 1]
```diff
@@ -501,68 +492,58 @@ class LiveConf {
...
     for(let i = 0; i < Object.keys(audioStreams).length; i ++ ) {
-      let audioLadderSpec = {...LadderSpecAudio};
+      let audioLadderSpec = {};
       const audioIndex = Object.keys(audioStreams)[i];
       const audio = audioStreams[audioIndex];
-      audioLadderSpec.bit_rate = audio.recordingBitrate;
+
+      // Find ladder rung for the specific channel layout (2 or 6 channels)
+      for (let i = 0; i < ladderProfile.audio.length; i ++) {
+        let elem = ladderProfile.video[i];
```
Comment (elv-gerald) [inline src/client/LiveConf.js:529]: Did you mean use `ladderProfile.audio[i];` here?

File: src/client/LiveConf.js [hunk 2]
```diff
@@ -501,68 +492,58 @@ class LiveConf {
...
+        if (elem.channels == audio.recordingChannels) {
+          audioLadderSpec = elem;
+          break;
+        }
+      }
+      if (audioLadderSpec == {}) {
+        // If no channels layout match, just use the first element in the ladder
+        audioLadderSpec = ladderProfile.video[0];
```
Comment (elv-gerald) [inline src/client/LiveConf.js:537]: Did you mean to use `ladderProfile.audio[0];` here?

File: src/client/LiveConf.js (hunk 1 above)
Comment (elv-serban) [reply src/client/LiveConf.js]: Fixed!

File: src/client/LiveConf.js (hunk 2 above)
Comment (elv-serban) [reply src/client/LiveConf.js]: Fixed!
Also fixed the 'codecs' which was overwritten here by mistake.

File: src/client/LiveConf.js [hunk 3]
```diff
@@ -74,6 +69,7 @@ const LiveconfTemplate = {
         part_ttl: 3600,
         playout_type: "live",
         source_timescale: null,
+        reconnect_timeout: 600,
```
Comment (elv-zenia) [inline src/client/LiveConf.js:72]: It looks like your `develop` branch was a little outdated. The reconnect_timeout stuff is in the latest

File: src/client/LiveConf.js [hunk 4]
```diff
@@ -459,6 +445,10 @@ class LiveConf {
       conf.live_recording.recording_config.recording_params.xc_params.connection_timeout = customSettings.connection_timeout;
     }
 
+    if(customSettings.reconnect_timeout) {
+      conf.live_recording.recording_config.recording_params.reconnect_timeout = customSettings.reconnect_timeout;
+    }
+
```
Comment (elv-zenia) [inline src/client/LiveConf.js:451]: This is in the latest too

**Review by elv-zenia — APPROVED** [review]

**Pull request merged**